engine/api/src/main/scala/pl/touk/nussknacker/engine/api/deployment/simple/SimpleStateStatus.scala

```diff
@@ -14,4 +14,5 @@ object SimpleStateStatus {
   val Canceled: StoppedStateStatus = StoppedStateStatus("CANCELED")
   val Failed: ErrorStateStatus = ErrorStateStatus("FAILED")
   val Error: ErrorStateStatus = ErrorStateStatus("ERROR")
+  val RunningError: ErrorStateStatus = ErrorStateStatus("RUNNING_ERROR")
```

Comment:
Maybe better name will be `RuninngWithError` or `RuninngWithWarning`? Because now it looks like it is problem during run like process was deployed but can't run.

Reply:
I've changed it to RunningWithError.